### ISSUE TITLE
Training Dataset interpretation

### ISSUE THREAD

**Issue by LMolr** [Arseha/peakonly#14]
Hello,
first of all **thank you** for the great paper and implementation.

I have some questions regarding the code and Training Data w.r.t. the informations I find in the Paper (and the "Supporting Informations" document).

In the training data i see for instance:
```json
{
    "code": "OD_003_1212",
    "label": 1,
    "number of peaks": 2,
    "peaks' labels": [1, 3],
    "borders": [[2, 34], [36, 58]],
    "description": "OriginalData_003",
    "rt": [23.287743333333, 23.908208333333],
    "scan": [1709, 1783],
    "intensity": [...]
}
```

I understand that the `border` field contains the boundaries for each peak.

## Peaks Labels

**Q**: What does the `peaks' labels` field indicates?

I'm asking this because the paper states that there are 3 kinds of labels for peak classification:
- noise
- 1 or more peak
- uncertain peaks

, but i see that this field has 4 different values (`0, 1, 2, 3`) across the dataset.

**Q**: I expected to see this field used during the training of the Classifier, but from the code it looks like it's not the case. Could you please clarify this?

## Label

**Q**: Does the `label` field indicate the absence (`0`) or presence (`1`) of at least one peak in the sample?
**Q**: Is it derived-from or related-to just the `peaks' labels` field?

## RT and Scan

**Q**: Are these fields derived from other fields? How?

Thanks in advance for your time.

**Comment by Arseha** [comment]
Hello, 
really sorry for the late answer. 

Borders
--------

> I understand that the border field contains the boundaries for each peak.

Yes, you are totally right about it.

Peaks Labels
-------------

It's an auxiliary field that is not currently used. I don't think that it will be used in the near future either. 0 - N/A, 1 - Good (smooth, high intensive), 2 - Low intensive (close to LOD), 3 - Lousy (not good), 4 - Noisy, strange (probably chemical noise). 

Label
-----

> Q: Does the label field indicate the absence (0) or presence (1) of at least one peak in the sample?

A: Yes, you are right here too.

> Q: Is it derived-from or related-to just the peaks' labels field?

A: There is only one relation. If the label is 0, peaks' labels should be empty.

RT and Scan
------------

> Q: Are these fields derived from other fields? How?

A: These fields are only needed to keep information about from which part of the original *.mzML file this signal was obtained (to avoid obtaining duplicates and etc.). So it is also auxiliary information.

**Comment by LMolr** [comment]
Hello, 
sorry, but I have been missing the notification about your reponse for months! :(

Thank you for your answers!
